src/utils/convert-filter.js

```diff
@@ -10,7 +10,7 @@ const convertFilter = (filter) => {
     switch (property.type()) {
     case 'string':
       return {
-        [property.name()]: { [Op.like]: `%${escape(value)}%` },
+        [property.name()]: { [Op.iLike]: `%${escape(value)}%` },
```

Comment:
I think this is just for postgres. Can you confirm it is working with MySQL?

Reply:
You are right, it's PG only. I am looking into other possible solutions now, preferably using something like "lower(column) LIKE %value%"